src/app/frontend/common/components/labels/labels.scss

```diff
@@ -0,0 +1,31 @@
+// Copyright 2015 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+@import '../../../variables';
+
+$label-height: $caption-font-size-base + 6;
+$label-margin: 0 $baseline-grid $baseline-grid 0;
+
+// Style taken(mostly) from angular material md-chip style.
+.kd-labels {
+  background-color: $body;
+  border-radius: $label-height / 2;
+  color: $muted;
```

Comment:
I was thinking about this. Pure black seemed a bit too distracting over the label background. Font color has to be set anyway because it will be overriden by other components if it's not set. Maybe something in the middle like `#444`?